handlers/telegram/telegram.go

```diff
@@ -187,9 +187,14 @@ func (h *handler) SendMsg(ctx context.Context, msg courier.Msg) (courier.MsgStat
 		return nil, fmt.Errorf("invalid auth token config")
 	}
 
+	attachments, err := handlers.ResolveAttachments(ctx, h.Backend(), msg.Attachments(), nil, true)
```

Comment:
I can't find a list of supported types for telegram.. so this will return all attachments regardless of type